### PR TITLE
TBD-8804: fixed

### PR DESCRIPTION
**Pull request by mtuhai**
**Please check if the PR fulfills these requirements**

- [X] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [X] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [X] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [X] Docs have been added / updated (for bug fixes / features) ?
- [X] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TBD-8804

**What is the new behavior?**
Fixed

### CODE REVIEW

**Comment by amedvedenko** [comment]
I don't see all jars you changed (except slf4j-api-1.7.25) in talend-update repository. Should they be added?

File: main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml [hunk 1]
```diff
@@ -468,13 +468,13 @@
             context="plugin:org.talend.libraries.hadoop.mapr.6.1.0" 
             id="json-20090211.jar" 
             name="json-20090211.jar"
-            mvn_uri="mvn:org.talend.libraries/json-20090211/6.0.0">
+            mvn_uri="mvn:org.talend.libraries/json/20090211">
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml]: I had modified this one myself after confirmation from UP. Why the change?

**Comment by mtuhai** [comment]
> I don't see all jars you changed (except slf4j-api-1.7.25) in talend-update repository. Should they be added?

Yes, all jars which are not exists in nexus/mvn repo should be added
